tests/test_plotting.py

```diff
@@ -0,0 +1,99 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import tempfile
+import os
+
+import lsst.utils.tests
+import lsst.utils
+
+from lsst.summit.utils.plotting import plot
+from lsst.summit.utils.utils import detectObjectsInExp
+import lsst.afw.image as afwImage
+from lsst.afw.geom import SpanSet
+from lsst.afw.detection import Footprint
+
+
+class PlottingTestCase(lsst.utils.tests.TestCase):
+    afwDataDir = lsst.utils.getPackageDir('afwdata')
```

Comment:
You can follow the pattern [here](https://github.com/lsst/pipe_tasks/blob/main/tests/test_quickFrameMeasurement.py) to check that the package is available, and only run the tests if they are, but you can't rely on this always being available.

Reply:
Check id afwDataDir exists added to code.

Reply:
Looks good, thanks!